packages/app-types/index.d.ts

```diff
@@ -112,6 +112,8 @@ export interface FirebaseNamespace {
   // Sets log handler for all Firebase components.
   onLog(logCallback: LogCallback, options?: LogOptions): void;
 
+  toJSON(): object;
```

Comment:
You should also modify the Firestore and the Firebase types if you modify this (firestore-types/index.d.ts and firebase/index.d.ts)

Reply:
done.